DNN Platform/Library/Application/DotNetNukeContext.cs

```diff
@@ -45,30 +44,27 @@ public static DotNetNukeContext Current
         {
             get
             {
-                if (_current == null)
+                if (current == null)
                 {
-                    _current = new DotNetNukeContext();
+                    current = Globals.DependencyProvider.GetRequiredService<IDnnContext>();
```

Comment:
Should we deprecate this function and ask consumers to use Dependency Injection? I decided not to deprecate this function because I didn't add the remaining members to the new `IDnnContext`. I think once we add the remaining members we can deprecate this method and recommend Dependency Injection.

If reviewers are in agreement I can create a work item